common/changes/@itwin/core-backend/affanK-fix-test-sensitive-to-order_2023-11-03-19-45.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/core-backend",
+      "comment": "Fixed test broken by updating SQLite version.",
```

Comment:
I think these change files end up in our changelogs / release notes at some point. Could you change the comment to empty string?

Reply:
done